nautobot/dcim/models/device_components.py

```diff
@@ -42,7 +43,7 @@
 )
 
 
-class ComponentModel(BaseModel, CustomFieldModel, RelationshipModel):
+class ComponentModel(BaseModel, CustomFieldModel, RelationshipModel, ComputedFieldModelMixin):
```

Comment:
Mixins should always precede bases (on the left).

Reply:
See comment below: https://github.com/nautobot/nautobot/pull/577#discussion_r657413204